massifs/logdircache.go

```diff
@@ -0,0 +1,644 @@
+package massifs
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/datatrails/go-datatrails-common/logger"
+)
+
+var (
+	ErrLogFileNoMagic                       = errors.New("the file is not recognized as a massif")
```

Comment:
NoMagic?

Reply:
Oh that is a tradition, files with bytes at the begining which are used to heuristically determin type. Those bytes are refered to as "magic" bytes

Reply:
ahh fantastic :) thanks, learnt something new!